engine/apps/webhooks/tasks/trigger_webhook.py

```diff
@@ -244,5 +265,24 @@ def execute_webhook(webhook_pk, alert_group_id, user_id, escalation_policy_id):
             escalation_error_code=error_code,
         )
 
-    if exception:
+    if isinstance(exception, EXECUTE_WEBHOOK_EXCEPTIONS_TO_MANUALLY_RETRY):
+        msg_details = (
```

Comment:
_nit_ label_with_underscore=value gives nice default behavior for logfmt in loki.